docs/spec/_ics/ics-030-signed-messages.md

```diff
@@ -87,7 +91,7 @@ to the Cosmos chain identifier. The user-agent should **refuse** signing if the
 `@chain_id` field does not match the currently active chain! The `@type` field
 must equal the constant `"message"`. The `@type` field corresponds to the type of
 structure the user will be signing in an application. For now, a user is only
-allowed to sign bytes of valid ASCII text ([see here](https://github.com/tendermint/tendermint/blob/master/libs/common/string.go#L61-L74)).
+allowed to sign bytes of valid ASCII text ([see here](https://github.com/tendermint/tendermint/blob/master/libs/common/string.go#L61-L74)). <!-- this is a dead link -->
```

Comment:
not sure what link to use as original is dead

Reply:
https://github.com/cometbft/cometbft/blob/main/libs/strings/string.go